lib/signaling/v2/remotetrackpublication.js

```diff
@@ -24,6 +24,7 @@ class RemoteTrackPublicationV2 extends RemoteTrackPublicationSignaling {
    */
   update(track) {
     this.enable(track.enabled);
+    this.setPriority(track.priority);
```

Comment:
what happens when 
a. Bob publishes track at low
b. alice changes subscriber priority of the track to medium
c. bob changes publish priority to medium - Does this cause `publishPriorityChanged`?
add a test for this.

Reply:
subscriber priority and publish priority for a remote track are kept separately. 
`RemoteTrackPublication.publishPriority` = the priority of the published RemoteTrack set by the RemoteParticipant
`RemoteTrack.priority` = the priority of the remote track set by subscriber.

since they are separate - you do get notified if published priority changes (even if its same as the subscribe priority set by the user for the same track )